Northstar.Client/mod/scripts/vscripts/ui/panel_mainmenu.nut

```diff
@@ -926,7 +926,7 @@ void function SpotlightButton_Activate( var button )
 	else
 	{
 		// discord links don't work in origin overlay
-		if ( link.find( "https://discord.gg" ) == 0 || link == "https://northstar.tf/discord" || link == "https://northstar.tf/wiki" )
+		if ( link.find( "https://discord.gg" ) == 0 || link == "https://northstar.tf/discord" )
```

Comment:
What's with this change? ^^

Reply:
I don't know, I didn't touch that, it may be an outdated version of the file where it didn't include the wiki, ignore this change

Reply:
resolved the outdated code!